e2e-tests/verify-upgradability.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+set -Eexuo pipefail
+
+# Run dfx stop if we run into errors and remove newest release wasm.
+trap "dfx stop & rm newest_release.wasm" EXIT SIGINT
+
+# Get newest release download url
+NEWEST_RELEASE="$(curl -s https://api.github.com/repos/dfinity/bitcoin-canister/releases/latest | grep "browser_download_url" | awk '{ print $2 }' | sed 's/,$//' | sed 's/"//g')"
+MANAGEMENT_CANISTER="aaaaa-aa"
+
+
+# Get newest release
+wget -O newest_release.wasm.gz "${NEWEST_RELEASE}"
+
+gunzip newest_release.wasm.gz
```

Comment:
I don't think you need to unzip, as the replica does this for you automatically (i.e. you can install the wasm.gz file directly).